### PR TITLE
IBP-4052 new germplasm details popup

### PR DESCRIPTION
**Pull request by abatac**

### CODE REVIEW

File: src/main/java/org/generationcp/breeding/manager/listimport/listeners/GidLinkClickListener.java [hunk 1]
```diff
@@ -95,45 +91,32 @@ private void openDetailsWindow(final Component component) {
...
-		ExternalResource germplasmBrowserLink = null;
-		if (tool == null) {
-			germplasmBrowserLink =
-					new ExternalResource(WorkbenchAppPathResolver.getFullWebAddress(DefaultGermplasmStudyBrowserPath.GERMPLASM_BROWSER_LINK
-							+ this.gid, "?restartApplication" + addtlParams));
-		} else {
-			germplasmBrowserLink =
-					new ExternalResource(WorkbenchAppPathResolver.getWorkbenchAppPath(tool, this.gid, "?restartApplication" + addtlParams));
-		}
+		final ExternalResource germplasmDetailsLink = new ExternalResource(
```
Comment (darla-leafnode) [inline src/main/java/org/generationcp/breeding/manager/listimport/listeners/GidLinkClickListener.java:94]: Can we extract this code to utility (passing gid, and contextUtil) so we don't form in multiple places in Vaadin:
```
final ExternalResource germplasmDetailsLink = new ExternalResource(
			WorkbenchAppPathResolver.getFullWebAddress(DefaultGermplasmStudyBrowserPath.GERMPLASM_DETAILS_LINK + this.gid + "?cropName="
				+ this.contextUtil.getProjectInContext().getCropType().getCropName() + "&programUUID=" + this.contextUtil
				.getCurrentProgramUUID()) + "&modal");
```

File: src/main/java/org/generationcp/ibpworkbench/GermplasmStudyBrowserApplication.java [hunk 2]
```diff
@@ -126,108 +129,123 @@ public Window getWindow(String name) {
...
 			} else if (GermplasmStudyBrowserApplication.AWHERE_WINDOW_NAME.equals(name)) {
-				Window awhereWindow = new Window("AWhere Test Tool");
+				final Window awhereWindow = new Window("AWhere Test Tool");
 				awhereWindow.setName(GermplasmStudyBrowserApplication.AWHERE_WINDOW_NAME);
 				awhereWindow.addComponent(new AWhereFormComponent());
 				awhereWindow.setWidth("100%");
 				awhereWindow.setHeight("100%");
 				this.addWindow(awhereWindow);
 				return awhereWindow;
+			} else if ("pedigree-details".equals(name)) {
```
Comment (darla-leafnode) [inline src/main/java/org/generationcp/ibpworkbench/GermplasmStudyBrowserApplication.java:224]: Extract "pedigree-details" as constant

File: src/main/java/org/generationcp/breeding/manager/listimport/listeners/GidLinkClickListener.java (hunk 1 above)
Comment (abatac) [reply src/main/java/org/generationcp/breeding/manager/listimport/listeners/GidLinkClickListener.java]: This is done.

File: src/main/java/org/generationcp/ibpworkbench/GermplasmStudyBrowserApplication.java (hunk 2 above)
Comment (abatac) [reply src/main/java/org/generationcp/ibpworkbench/GermplasmStudyBrowserApplication.java]: This is done.

**Pull request merged**